modules/apps/blogs/blogs-web/package.json

```diff
@@ -1,6 +1,8 @@
 {
 	"dependencies": {
+		"@liferay/frontend-js-state-web": "*",
 		"frontend-js-web": "*",
+		"item-selector-taglib": "*",
```

Comment:
`blogs-web` depends on `item-selector-taglib` because the latter "owns" the state related to the cover image.